include/NAS2D/Configuration.h

```diff
@@ -85,9 +89,9 @@ class Configuration
 
 	bool readConfig(const std::string& filePath);
 
-	void parseGraphics(void *node);
-	void parseAudio(void *node);
-	void parseOptions(void *node);
+	void parseGraphics(NAS2D::Xml::XmlElement *node);
+	void parseAudio(NAS2D::Xml::XmlElement *node);
+	void parseOptions(NAS2D::Xml::XmlElement *node);
```

Comment:
Consistency and convention.
Lines 74-76 have their reference ampersands on the left as well.
Move the asterisk to the left. These are types, not values that need to be dereferenced.

Reply:
Additionally, the implementation of these functions also has them on the left.

Reply:
I suppose technically that was pre-existing, and such a style change is outside the scope of this PR, though it may indeed be a good time to change it. A quick grep shows very few occurrences of this style.

Reply:
Actually, considering I changed the implementation file and put the `*` on the left there, I should probably be consistent about it in the header file too. Anyway, change has already been made.